src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -124,6 +124,16 @@ def which(binary):
     return None
 
 
+def check_cmd_test_hook_file(filename):
+    curr_dir = os.path.dirname(os.path.realpath(__file__))
+    test_hook_file = os.path.join(curr_dir, 'tests/latest/data', filename)
+    if os.path.exists(test_hook_file):
+        with open(test_hook_file, "r") as f:
+            return json.load(f)
```

Comment:
it's strange that one function to return two types of values.

Reply:
Good point, changed in commit [386f9d0](https://github.com/Azure/azure-cli/pull/19115/commits/386f9d05646beca7f620c78bab183b97e38a99fb). Now this function will only return string, the default value is an empty string.